src/content/cloud/deploy-from-helm.mdx

```diff
@@ -10,6 +10,8 @@ import Image from '@theme/Image';
 You can launch additional services for your development environments on Okteto Cloud using Helm repositories.
 Okteto automates all the processes needed to deploy, configure, upgrade, and destroy your Helm releases, so you can focus on building amazing applications.
 
+You can also [bring your own Helm repository](/docs/enterprise/administration/configuration/#applications) to the catalog and get the same one-click deployment experience for your own services when using [Okteto Self-Hosted](/docs/enterprise/).
```

Comment:
It was a bit confusing to me what "catalog" meant until I looked at the image below in the deploy section, how about something like this:

```suggestion
You can also [list your own repository under the **Helm Catalog**](/docs/enterprise/administration/configuration/#applications) tab of the **Deployment** menu. This would give you the same one-click deployment experience for your own services when using [Okteto Self-Hosted](/docs/enterprise/).
```